### ISSUE TITLE
Do not assume "master" is the default branch

### ISSUE THREAD

**Issue by ghostsquad** [jsonnet-bundler/jsonnet-bundler#122]
`jb install` should not assume that `master` is the default branch.

Additionally, if you _had_ to assume a default branch, this is request to change this to `main` to bring more awareness around the oppressive terminology. https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/

**Comment by metalmatze** [comment]
Agreed, we should just use whatever the upstream server has set as default.
Hopefully with that we can avoid making any assumptions on our side to begin with.

I wouldn't update just yet, and wait for what GitHub decides to do for all of GitHubs default branches, then adjust accordingly :)

**Comment by ghostsquad** [comment]
@metalmatze there's no need to wait for Github. Jsonnet can be a source of raised awareness. Force developers to pin to `@master` if they so choose.

**Comment by ghostsquad** [comment]
Github has made the change, and this problem has been reported again in #144
